src/history.rs

```diff
@@ -47,6 +44,9 @@ pub const MIN_HISTORY_SIZE_BYTES: usize = size_of::<DynamicHistory>()
     + MIN_CLOCKS_LEN * size_of::<LogicalClock>()
     + MIN_LOG_LEN * size_of::<CompactLogItem>();
 
+const EPOCH_WRAPAROUND_THRESHOLD_TOP: u16 = ProbeEpoch::MAX - 3;
```

Comment:
This offset is to order in the face of a rollover / restart I assume? Is this documented somewhere? If not, would you mind writing  comment here explaining what these are for?

Reply:
This is for handling epoch rollover. Will link you to the doc out of band.